wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder.swift

```diff
@@ -382,7 +383,11 @@ extension EventDecoder {
         return events.filter { event in
             // The only message we process arriving in the self conversation from other users is availability updates
             if event.conversationUUID == selfConversationID, event.senderUUID != selfUserID, let genericMessage = GenericMessage(from: event) {
-                return genericMessage.hasAvailability
+                let included = genericMessage.hasAvailability
+                if !included {
+                    WireLogger.updateEvent.warn("dropping stored event", attributes: [LogAttributesKey.eventId.rawValue: event.safeUUID])
```

Comment:
Maybe easier to read?
```suggestion
                    WireLogger.updateEvent.warn(
                        "dropping stored event",
                        attributes: [LogAttributesKey.eventId.rawValue: event.safeUUID]
                    )
```